src/providers/Asset.tsx

```diff
@@ -25,6 +25,7 @@ interface AssetProviderValue {
   title: string | undefined
   owner: string | undefined
   price: BestPrice | undefined
+  type: string | undefined
```

Comment:
We should be more specific here since we require a specific string. We can get this from ocean.js typings with `MetadataMain['type']`. Then it's also more future proof in case another `type` is added in ocean.js